src/components/panel/panel.js

```diff
@@ -355,6 +382,16 @@ angular
  * @param {!MdPanelPosition} position
  */
 
+/**
+ * @ngdoc method
+ * @name MdPanelRef#addToGroup
+ * @description
+ * Add a panel to a group, either automatically during its original creation, or
```

Comment:
Add -> Adds

Remove the "automatically during its original creation". We only care about the public API in this description. The users shouldn't have to care what happens behind the scenes during creation.